smartcosmos-framework/pom.xml

```diff
@@ -45,6 +45,10 @@
             <artifactId>spring-cloud-starter-oauth2</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-test</artifactId>
```

Comment:
I'd love to use `test` scope, but if we changed that, only **framework tests** would take the dependency into account, and it would not be available in **resource tests** in modules depending on Framework.